tests/lib/hardware/libvirt.py

```diff
@@ -365,21 +365,26 @@ def _create_network(self):
             dhcp_start = str(netaddr.IPAddress(network.first+2))
             dhcp_end = str(netaddr.IPAddress(network.last-1))
             xml = textwrap.dedent("""
-                <network>
-                <name>%(network_name)s</name>
-                <forward mode="nat"/>
-                <ip address="%(host_ip)s" netmask="%(netmask)s">
-                    <dhcp>
-                        <range start="%(dhcp_start)s" end="%(dhcp_end)s" />
-                    </dhcp>
-                </ip>
-                </network>
-            """ % {
+    <network xmlns:dnsmasq='http://libvirt.org/schemas/network/dnsmasq/1.0'>
+    <name>%(network_name)s</name>
+    <forward mode="nat"/>
+    <ip address="%(host_ip)s" netmask="%(netmask)s">
+        <dhcp>
+            <range start="%(dhcp_start)s" end="%(dhcp_end)s" />
+        </dhcp>
+    </ip>
+    <dnsmasq:options>
+        <dnsmasq:option value="address=/.local/%(registry_mirror)s"/>
```

Comment:
What is this good for? I don't get it.

Reply:
This is needed to have the DNS setup by libvirt to resolve your cache registries (docker.io.local, registry.suse.de.local etc...) to the IP of `registry_mirror_address`

I actually passes options to dnsmasq